generator/integration_tests/golden/database_admin_client.gcpcxx.pb.h

```diff
@@ -31,111 +31,538 @@ namespace cloud {
 namespace golden {
 inline namespace GOOGLE_CLOUD_CPP_NS {
 
+/**
+ * Cloud Test Database Admin API
+ *
+ * The Cloud Test Database Admin API can be used to create, drop, and
+ * list databases. It also enables updating the schema of pre-existing
+ * databases. It can be also used to create, delete and list backups for a
+ * database and to restore from an existing backup.
+ *
+ */
 class DatabaseAdminClient {
  public:
   explicit DatabaseAdminClient(ConnectionOptions const& options = ConnectionOptions());
   explicit DatabaseAdminClient(std::shared_ptr<DatabaseAdminConnection> connection);
   ~DatabaseAdminClient();
 
+  /**
+   * Lists Cloud Test databases.
+   *
+   * @param parent  Required. The instance whose databases should be listed.
```

Comment:
Do we need to include the "Required" comment here? I don't think we do that elsewhere in our other libraries, and it doesn't seem necessary given the signature of the C++ function.

... oh, hmm, I'm guessing that this is coming directly out of comments in a `.proto` file? Would any of them say "Optional" ever? If so, that might be strange.

I wonder if we might want to lop this required/optional part off.